### DISCUSSION TITLE
https://plugin-blog2.vuejs.press/zh/demo.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#619] [Announcements]
# https://plugin-blog2.vuejs.press/zh/demo.html

为了让你更好的了解如何使用插件，我们使用 vuepress-plugin-blog2 和 @vuepress/theme-default 制作了一个案例。 源代码 (https://github.com/vuepress-theme-hope/vuepress-theme-hope/tree/main/demo/blog2/); 预览 (https:/...

https://plugin-blog2.vuejs.press/zh/demo.html

**Comment by DillonHuang** [comment]
uuuuu
